src/csharp/Microsoft.Spark.E2ETest/UdfTests/BroadcastTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.UdfTests
+{
+    [Serializable]
+    public class BroadcastTestClass
+    {
+        public int intValue { get; set; }
```

Comment:
Can you explain what's up with the casing in this class? And why do the constructor parameter names have leading `_`s?

Reply:
Changed to follow c# variable naming convention.